### PR TITLE
560 Change whether school needs to order chromebooks

### PR DESCRIPTION
**Pull request by aldavidson**
### Context

In #367 we introduced a view-only UI for schools to check their details. This PR allows a school to change their Chromebook information. (Trello card [560](https://trello.com/c/l8h6foA9/560-check-your-school-details-change-whether-school-needs-to-order-chromebooks))

### Changes proposed in this pull request

* Add UI for schools to edit their chromebook information
* Move the `responsible_body/devices/chromebook_information_form` object back to the top level, as it is now used in two contexts

### Guidance to review

Screenshots:
Screen Shot 2020-09-02 at 14 01 25
Screen Shot 2020-09-02 at 14 01 07

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -146,6 +146,8 @@ en:
     what_to_do_with_the_vouchers: What to do with the vouchers
     not_offered_vouchers_yet: If you haven’t been offered vouchers yet
   school:
+    chromebooks:
+      success: We've saved your choices
```
Comment (benilovj) [inline config/locales/en.yml:150]:
```suggestion
      success: We’ve saved your choices
```

File: app/views/school/chromebooks/edit.html.erb [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+      <li>the school’s recovery email address</li>
+    </ul>
+    <p class="govuk-body">
+      We need this information so we can secure the devices.
+    </p>
+    <p class="govuk-body">
+      If you need help finding these details, email <%= govuk_link_to('COVID.TECHNOLOGY@education.gov.uk', 'mailto:COVID.TECHNOLOGY@education.gov.uk') %>
+    </p>
+
+    <%= form_for @chromebook_information_form, url: school_chromebooks_path, method: :patch do |f| %>
```
Comment (benilovj) [inline app/views/school/chromebooks/edit.html.erb:27]: How much of this is reusable/shared with the RB chromebook form?

File: config/locales/en.yml [hunk 3]
```diff
@@ -145,6 +146,8 @@ en:
...
```
Comment (tonyheadford) [inline config/locales/en.yml:150]: Minor content suggestion ?
```suggestion
      success: We’ve saved your choices
```

File: config/routes.rb [hunk 4]
```diff
@@ -98,6 +98,8 @@
   namespace :school do
     get '/', to: 'home#show', as: :home
     get '/details', to: 'details#show', as: :details
+    get '/chromebooks/edit', to: 'chromebooks#edit', as: :edit_chromebooks_details
```
Comment (benilovj) [inline config/routes.rb:101]: If you don't set the `as:` clause, you will get a `school_chromebooks_edit_path` generated for you automatically as per Rails conventions.

**Review by benilovj — APPROVED** [review]
Looks good, some minor points.

**Review by tonyheadford — APPROVED** [review]
Looks good, just one minor content suggestion

File: config/routes.rb (hunk 4 above)
Comment (aldavidson) [reply config/routes.rb:101]: thanks!

File: config/locales/en.yml (hunk 3 above)
Comment (aldavidson) [reply config/locales/en.yml]: @benilovj made the same suggestion, which I've committed - thanks!

File: app/views/school/chromebooks/edit.html.erb (hunk 2 above)
Comment (aldavidson) [reply app/views/school/chromebooks/edit.html.erb]: ooooh, as it turns out, all of it if I make one or two tweaks..... good call!

**Pull request merged**